tests/cases/conformance/constEnums/constEnum4.ts

```diff
@@ -0,0 +1,3 @@
+if (true) {
```

Comment:
You need to remove the curly braces to actually test your change

Reply:
sorry my fault